CRM/Mailing/Info.php

```diff
@@ -92,33 +92,31 @@ public function getAngularModules() {
       ),
     );
 
-    $config = CRM_Core_Config::singleton();
     $session = CRM_Core_Session::singleton();
     $contactID = $session->get('userID');
-    $civiMails = civicrm_api3('Mailing', 'get', array());
-    $campNames = civicrm_api3('Campaign', 'get', array());
-    $mailingabNames = civicrm_api3('MailingAB', 'get', array());
-    $mailStatus = civicrm_api3('MailingJob', 'get', array());
-    $groupNames = civicrm_api3('Group', 'get', array());
-    $headerfooterList = civicrm_api3('MailingComponent', 'get', array());
+
+    $params = array('options' => array('limit' => 0));
+    $civiMails = civicrm_api3('Mailing', 'get', $params);
```

Comment:
Would it be difficult to switch it to an entityRef?
Is there an angular directive for entityRef yet? Might be straightforward to add one.